inline_scan.sh

```diff
@@ -249,14 +249,27 @@ get_and_validate_images() {
 }
 
 prepare_inline_container() {
-    # Check if env var is overriding which inline-scan image to utilize.
-    if [[ -z "${SYSDIG_CI_IMAGE-docker.io/anchore/inline-scan:v0.6.1}" ]]; then
-        printf '\n%s\n' "Pulling ${INLINE_SCAN_IMAGE}"
-        docker pull "${INLINE_SCAN_IMAGE}"
+    # Retrieve dynamically from secure the Anchore version for compatibility reasons
+    if [[ -z "$INLINE_SCAN_IMAGE" ]]; then
+        printf 'Retrieving remote Anchore version from Sysdig Secure APIs\n'
+        SCANNING_ANCHORE_STATUS=$(curl -sSkf  -H "Authorization: Bearer ${SYSDIG_API_TOKEN}" "${SYSDIG_SCANNING_URL%%/}/anchore/status")
+        INLINE_SCAN_IMAGE_VERSION=$(echo ${SCANNING_ANCHORE_STATUS} | grep -o -E '"version":[ \t]?".*"' | awk -F ":" '{print $2}' | awk -F '"' '{print $2}')
```

Comment:
such an intuitive language to parse jsons 😉, nice idea using `-o`
the only thing I can think of is that maybe we want to bail early if `curl` had any issues (i.e. `$?` is nonzero)

Reply:
afaik, any command failure should exits immediately the script because of `set -eou pipefail`.

Reply:
Thanks, I missed that :)